python-flask-sdk/katanemo_flask/flask_auth.py

```diff
@@ -0,0 +1,65 @@
+import json
+import katanemo_identity
+import katanemo_auth
+import requests
+from flask import request as flask_request
+
+class KatanemoFlaskAuth:
+    
+    def __init__(self, app):
+        self.app = app
+        self.api_client = None
+        self.auth_client = None
+        self.api_access_token = None
+        self.client_id = None
+
+    def register(self, client_name, service_id, client_id, client_secret):
+        self.client_id = client_id
+        self.client_secret = client_secret
+        access_api_client = katanemo_identity.AccessControlApi(self._get_access_api_client())
+        req = katanemo_identity.OAuthTokenRequest(clientId=client_id, clientSecret=client_secret)
+        resp = access_api_client.get_short_term_token(req)
+        self.api_access_token = resp.access_token
+
+    def authorize_request(self, access_token, request_path, http_method, request_body):
+        auth_api_client = self._get_auth_client()
+
+        req = katanemo_auth.AuthorizationRequest(
+            token=access_token,
+            path=request_path,
+            http_method=http_method,
+            request_body=request_body,
+        )
+        auth_api_client.authorize_request(req)
+    
+    def authorize(self, redirect_uri, service_id):
+        authorizeUrl = "https://api.katanemo.com/authorize?state={}&service={}".format(redirect_uri, service_id)
+        resp = requests.get(authorizeUrl, allow_redirects=False)
```

Comment:
this is to prevent python's requests method to not follow http redirects -  I think we may not need this specific setting here but let me verify without passing this param

Reply:
looked at code - its for the following reason,

```
        if resp.status_code != 302:
            raise Exception("Failed to get login redirect url")
        login_redirect_url = resp.headers["Location"]

```

if we don't do `allow_directs=False` then python sdk will do the redirect and browser will not see it